src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3485,6 +3485,72 @@ private IOperation VisitNoneOperationExpression(IOperation operation)
             return Operation.CreateOperationNone(semanticModel: null, operation.Syntax, operation.ConstantValue, childrenBuilder.ToImmutableAndFree(), operation.IsImplicit);
         }
 
+        public override IOperation VisitInterpolatedString(IInterpolatedStringOperation operation, int? captureIdForResult)
+        {
+            foreach (IInterpolatedStringContentOperation element in operation.Parts)
+            {
+                if (element.Kind == OperationKind.Interpolation)
+                {
+                    var interpolation = (IInterpolationOperation)element;
+                    _evalStack.Push(Visit(interpolation.Expression));
+
+                    if (interpolation.Alignment != null)
+                    {
+                        _evalStack.Push(Visit(interpolation.Alignment));
+                    }
+
+                    if (interpolation.FormatString != null)
+                    {
+                        _evalStack.Push(Visit(interpolation.FormatString));
+                    }
+                }
+                else
+                {
+                    var interpolatedStringText = (IInterpolatedStringTextOperation)element;
+                    // PROTOTYPE(DATAFLOW): Should we avoid flow captures for constant string literal "Text" by not pushing onto the eval stack?
```

Comment:
>DATAFLOW [](start = 33, length = 8)

I think our convention is to use "dataflow" all low case, matches the branch name